pkg/graph/resolvers/model_planR.go

```diff
@@ -9,12 +9,22 @@ import (
 )
 
 // ModelPlanCreate implements resolver logic to create a model plan
-func ModelPlanCreate(logger *zap.Logger, plan *models.ModelPlan, store *storage.Store) (*models.ModelPlan, error) {
+func ModelPlanCreate(logger *zap.Logger, plan *models.ModelPlan, store *storage.Store, principalInfo *models.UserInfo) (*models.ModelPlan, error) {
 
 	createdPlan, err := store.ModelPlanCreate(logger, plan)
 	if err != nil {
 		return nil, err
 	}
+	///TODO get user full name, need to acces CEDAR for this
+	colab := models.PlanCollaborator{
+		EUAUserID:   *createdPlan.CreatedBy,
+		ModelPlanID: createdPlan.ID,
+		TeamRole:    models.TeamRoleModelLead,
+		CMSCenter:   models.CMSCMMI,
+		FullName:    principalInfo.CommonName, //TOOD get this information from CEDAR
+	}
```

Comment:
Can be extracted to method

Reply:
it could, but it's not being reused at this point. I don't think it needs to be here, I think this is probably more readable in this instance.